AndroidStudio-Flutter-setup.md

```diff
@@ -58,14 +58,28 @@ flutter doctor
 it should now tell all (but maybe chrome) is OK.
 
 # RS mobile: Prepare project
+## Necessary packages
+RS-mobile depends on the following packages to be avail on path-level parallel to RS-mobile itself and with a name actually not matching the their repo names:
+{as of rs-mobile: SHA-1: 8a16a84711388872d455ce2a81dbfe4f8643f63f}
+
+- Repo: https://gitlab.com/Kumaravinash9/retroshare-dart-see.git
+  1. clone it to the same folder-hierarchy level as you will do for rs-mobile
```

Comment:
In future we will update the ref of it. So after that we don't have to clone these repo separately

Reply:
at this future time the section in howto will get also removed.

Reply:
okk please make a PR for this on my PR i will merge this. Thnaks

Reply:
assuming "my PR" is a shortcut for your Git-Repo.

Reply:
PR on your Reps-master done.